apps/web/vibes/2px/examples/radio-button.tsx

```diff
@@ -0,0 +1,11 @@
+import RadioButton from '@/vibes/2px/components/radio-button'
+
+export default function Preview() {
+  return (
+    <div className="flex min-h-48 flex-col items-start justify-center gap-3 bg-background p-5 sm:min-h-64 sm:p-8 lg:min-h-80 lg:p-12">
+      <RadioButton label="Radio item" />
```

Comment:
In this example, I'm able to select this radio button, but not unselect it later. I think it'd be nice to have that working both ways as an example.

Reply:
A radio button can't be unselected @freadbaker, checkboxes on the other hand can be unselected.

Reply:
Please check now @freadbaker, I made all of them the same name so they disable each other.

Reply:
That's working great! Sorry for the confusion earlier.